x-pack/plugins/uptime/common/constants/alerts.ts

```diff
@@ -8,16 +8,22 @@
 import { ActionGroup } from '../../../alerting/common';
 
 export type MonitorStatusActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.monitorStatus'>;
-export type TLSActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.tls'>;
+export type TLSLegacyActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.tls'>;
+export type TLSActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.tlsIndividual'>;
 export type DurationAnomalyActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.durationAnomaly'>;
 
 export const MONITOR_STATUS: MonitorStatusActionGroup = {
   id: 'xpack.uptime.alerts.actionGroups.monitorStatus',
   name: 'Uptime Down Monitor',
 };
 
-export const TLS: TLSActionGroup = {
+export const TLS_LEGACY: TLSLegacyActionGroup = {
   id: 'xpack.uptime.alerts.actionGroups.tls',
+  name: 'Uptime TLS Alert (Legacy)',
+};
+
+export const TLS: TLSActionGroup = {
+  id: 'xpack.uptime.alerts.actionGroups.tlsIndividual',
```

Comment:
`tlsIndividual` seems awkward. imagine once we get rid of tls legacy. this name will look odd.

i would prefer something like 'tlsCertificate' `tlsRule`